project/Settings.scala

```diff
@@ -57,7 +57,7 @@ object Settings {
         from("openjdk:8")
         expose(8000)
         add(artifact, artifactTargetPath)
-        entryPoint("java", "-jar", artifactTargetPath)
+        entryPoint("/bin/bash", "-c", "java -jar " + artifactTargetPath + " $CROMWELL_ARGS")
```

Comment:
i spent a few hours banging against this, and I couldn't find a better way.  Docker is pretty clear that if you want shell variables to be used, you have to launch a shell yourself.